csrc/fused_dense_cuda.cu

```diff
@@ -79,6 +103,30 @@ cublasStatus_t gemm_bias(
     const float* beta,
     float* C,
     int ldc) {
+#ifdef USE_ROCM
```

Comment:
Same comment, is this required?

Reply:
Hipify converts datatype CUDA_R_xxF to  HIP_R_xxF but what we need is  HIPBLA_R_xxF. For that reason, this is needed.